src/ui/AppLayout.tsx

```diff
@@ -0,0 +1,13 @@
+import { Layout, LayoutProps } from "react-admin";
+import { ReactQueryDevtools } from "react-query/devtools";
+import { AppBar } from "./AppBar";
+import { isEnvModeDev } from "../utils";
+
+export const AppLayout = (props: LayoutProps) => {
+  return (
+    <>
+      <Layout {...props} appBar={AppBar} />
+      {isEnvModeDev ? <ReactQueryDevtools /> : null}
```

Comment:
ReactDevTools are already hidden in production. https://tanstack.com/query/v3/docs/react/devtools#import-the-devtools